### DISCUSSION TITLE
Is it possible to support flatbuffer 2.0?

### DISCUSSION THREAD

**Discussion by littleday** [onnx/tensorflow-onnx#1841] [Ideas]
Hi team, we just started to use onnx models to speed up our model performance. However, the infrastructure requires flatbuffer 2.0. Is it possible to support it in the next release?

**Comment by hwangdeyu** [comment]
Was the flatbuffer 2.0 blocked you? I can use tf2onnx successfully in `flatbuffer  2.0`.

**Reply by ahoereth** [reply]
`setup.py` seems to [pin flat buffers to 1.12](https://github.com/onnx/tensorflow-onnx/blob/f4902a42382eb127d3b8e4a5351fed1df7968910/setup.py#L85) , as of #1511.

**Comment by bryce-evans** [comment]
The latest `tensorflow-onnx` throws errors when used with the latest `tensorflow`. For many projects, this is a non-starter.

It's been 18 months since the offending code was committed and 6+ months with no response from the team. Is this going to be handled (i.e. modifying the pin at `setup.py`) at some point?

**Comment by ahoereth** [comment]
Looks tensorflow-onnx now installs flatbuffers 2.0, pinned to `flatbuffers>=1.12` since #2050 and to `flatbuffers<3.0,>=1.12` on current main as of #2057.
